src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -818,10 +845,10 @@ public void validateStaticCodeAnalysisSettings(ProgrammingLanguageFeature progra
     }
 
     /**
-     * Validates settings for exercises, where allowManualFeedbackRequests is set
+     * Validates settings for exercises, where allowFeedbackRequests is set
      */
-    public void validateManualFeedbackSettings() {
-        if (!this.getAllowManualFeedbackRequests()) {
+    public void validateFeedbackSettings() {
```

Comment:
I think the renaming here makes it less clear what this method does. It seems like it validates the settings for feedbacks, when it actually validates if a feedback request can be made. Maybe call it `validateSettingsForFeedbackRequest`?